backend/src/main/java/com/woowacourse/moragora/service/MeetingService.java

```diff
@@ -83,23 +93,25 @@ public MeetingResponse findById(final Long meetingId, final Long userId) {
     // TODO update (1 + N) -> 최적하기
     // TODO 출석 제출할 때 구현 예정
     @Transactional
-    public void updateAttendance(final Long meetingId, final UserAttendancesRequest requests) {
-/*
+    public void updateAttendance(final Long meetingId, final Long userId, final UserAttendanceRequest request) {
         final Meeting meeting = findMeeting(meetingId);
-        meeting.increaseMeetingCount();
-
-        final List<Long> absentUserIds = requests.getUsers().stream()
-                .filter(UserAttendanceRequest::getIsTardy)
-                .map(UserAttendanceRequest::getId)
-                .collect(Collectors.toList());
-
-        final List<Attendance> attendances = attendanceRepository.findByMeetingId(meetingId);
-        for (final Attendance attendance : attendances) {
-            if (absentUserIds.contains(attendance.getUser().getId())) {
-                attendance.increaseTardyCount();
-            }
+
+        final LocalTime entranceTime = meeting.getEntranceTime();
+        final boolean isExcess = serverTime.isExcessClosingTime(LocalTime.now(), entranceTime);
+        if (isExcess) {
+            throw new ClosingTimeExcessException();
         }
-*/
+
+        final List<Participant> participants = participantRepository.findByMeetingId(meeting.getId());
+        final Participant participant = participants.stream()
```

Comment:
```suggestion
        final Participant participant  = participantRepository.findByMeetingIdAndUserId(meetingId, userId)
              .orElseThrow(ParticipantNotFoundException::new)
```
Optional을 이용해서 이렇게 바꿔볼 수도 있지 않을까요??

Reply:
넵 저도 이 방법을 생각해보았는데요!! 
DB에서 필터링하는게 좋을까? 리스트를 불러와 Java에서 필터링하는게 좋을까? 고민하다가 Java에서 필터링하는 방법을 선택했어요!! 
아직 어떤 방법이 더 성능은 좋을지는 알아보지 않았는데 이번 기회에 알아보겠습니다!!

Reply:
<img width="620" alt="image" src="https://user-images.githubusercontent.com/67885363/179711834-a7c7d554-1406-44d4-ae57-815a50174687.png">
2가지 방법으로 시간을 비교해봤는데요!! SQL문으로 조회해오는게 훨씬 빠르네요!! SQL문으로 변경하겠습니다

Reply:
와... 11배 빨라졌네요 ㅎㅎ